tldw_Server_API/app/core/config.py

```
@@ -3526,6 +3526,45 @@ def _get_str(section: str, key: str, default: str | None) -> str | None:
+            except _CONFIG_NONCRITICAL_EXCEPTIONS:
+                return None
+
+        # Parakeet MLX settings
+        mlx_model_id = _get_str('STT-Settings', 'mlx_model_id', 'mlx-community/parakeet-tdt-0.6b-v3') or 'mlx-community/parakeet-tdt-0.6b-v3'
+        mlx_cache_dir = _get_str('STT-Settings', 'mlx_cache_dir', '') or ''
+        mlx_chunk_duration = _get_float('STT-Settings', 'mlx_chunk_duration', 30.0)
+        mlx_overlap_duration = _get_float('STT-Settings', 'mlx_overlap_duration', 5.0)
+        buffered_chunk_duration = _get_float('STT-Settings', 'buffered_chunk_duration', mlx_chunk_duration)
+        buffered_total_buffer = _to_optional_float(_get_str('STT-Settings', 'buffered_total_buffer', None))
+        buffered_merge_algo = _get_str('STT-Settings', 'buffered_merge_algo', 'middle') or 'middle'
+        streaming_fallback_to_whisper = _get_bool('STT-Settings', 'streaming_fallback_to_whisper', True)
```

Ruff (0.15.1)

[error] Boolean positional value in function call

(FBT003)